CONTRIBUTING.md

```diff
@@ -0,0 +1,155 @@
+- Determine which repository the problem should be reported in.
+- Perform a cursory search to see if the problem has already been reported. If it has and the issue is still open, add a comment to the existing issue instead of opening a new one.
+- Provide a step-by-step description of the suggested enhancement in as many details as possible.
+- Provide specific examples to demonstrate the steps. Include copy/pasteable snippets (when applicable) which you use in those steps, as Markdown code blocks.
+- Describe the current behaviour and explain which behaviour you expected to see instead and why.
+- Include screenshots and/or animated GIFs which help you demonstrate the steps or point out the part of Pure Finance which the issue is related to. You can use this tool to record GIFs on macOS and Windows, and [this tool](https://github.com/GNOME/byzanz) on Linux.
+- Specify the name and version of the OS/Browser/other applicable software you're using.
+
+Then, you can file a new feature by using a template:
+[Issue Template](#)
+
+### 4. Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion for Pure Finance, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion and find related suggestions.
+
+Note: Please don't file an issue to ask a question. You'll get faster results by using the resources below.
+
+Before submitting an enhancement suggestion, check if there is already a filed one which provides that enhancement.
+
+First of all, you can validate it with the community, our official channels are:
+
+- [Telegram group](https://t.me/vesperfinance/)
+- [Discord](https://discord.com/invite/Pm8UDaazsK)
+
+How Do I Submit A (Good) Enhancement Suggestion?
+
+General tips:
+
+- Use a clear and descriptive title to identify the suggestion.
+- Provide a step-by-step description of the suggested enhancement in as many details as possible.
+- Provide specific examples to demonstrate the steps. Include copy/pasteable snippets which you use in those examples, as Markdown code blocks.
+- Describe the current behaviour and explain which behaviour you expected to see instead and why.
+- Include screenshots and/or animated GIFs which help you demonstrate the steps or point out the part of Pure Finance which the suggestion is related to. You can use this tool to record GIFs on macOS and Windows, and [this tool](https://github.com/GNOME/byzanz) on Linux.
+- Explain why this enhancement would be useful to most Pure Finance users.
+- Specify the name and version of the OS/Browser/other applicable software you're using.
+
+Then, you can file a new feature by using a template:
+[Feature Template](#)
+
+- Github Feature Request Template
+  [Feature Request Template](#)
```

Comment:
Is this OK? Previous sections only have one link. Seems like a copy/paste/edit error.